src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,270 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h, hr } from 'react-hyperscript-helpers'
+import ReactJson from 'react-json-view'
+import Select from 'react-select'
+import { ButtonPrimary, ButtonSecondary } from 'src/components/common'
+import ErrorView from 'src/components/ErrorView'
+import { icon } from 'src/components/icons'
+import { TextInput } from 'src/components/input'
+import { breadcrumbHistoryCaret } from 'src/components/job-common'
+import Modal from 'src/components/Modal'
+import { Ajax } from 'src/libs/ajax'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+
+
+const CallCacheWizard = ({
+  onDismiss, workflowId, callFqn, attempt, index
+}) => {
+  /*
+   * State setup
+   */
+
+  const [otherWorkflowId, setOtherWorkflowId] = useState()
+  const [otherWorkflowMetadata, setOtherWorkflowMetadata] = useState()
+  const [otherCallFqn, setOtherCallFqn] = useState()
+  const [otherIndex, setOtherIndex] = useState()
+  const [otherAttemptSucceeded, setOtherAttemptSucceeded] = useState()
+  const [otherCallSelected, setOtherCallSelected] = useState(false)
+  const [diff, setDiff] = useState()
+  const [metadataFetchError, setMetadataFetchError] = useState()
+  const [diffError, setDiffError] = useState()
+
+  const signal = Utils.useCancellation()
+
+  /*
+   * Data Fetchers
+   */
+
+  const readCalls = async otherWf => {
+    try {
+      const includeKey = [
+        'end', 'start', 'executionStatus'
+      ]
+      const excludeKey = []
+      const wf = await Ajax(signal).CromIAM.workflowMetadata(otherWf, includeKey, excludeKey)
+      setOtherWorkflowMetadata(wf)
+    } catch (error) {
+      console.log(error)
+      if (error instanceof Response) setMetadataFetchError(await error.text())
+      else if (_.isObject(error)) setMetadataFetchError(JSON.stringify(error))
+      else setMetadataFetchError(error)
+    }
+  }
+
+  const fetchDiff = async (otherWf, otherCall, otherIx) => {
+    try {
+      const diff = await Ajax(signal).CromIAM.callCacheDiff(workflowId, callFqn, Number(index), otherWf, otherCall, Number(otherIx))
+      setDiff(diff)
+    } catch (error) {
+      console.log(error)
+      if (error instanceof Response) setDiffError(await error.text())
+      else if (_.isObject(error)) setDiffError(JSON.stringify(error))
+      else setDiffError(error)
+    }
+  }
+
+  const otherCallFqnSelectionOptions = _.flow(
+    _.keys,
+    _.map(name => { return { value: name, label: name } })
```

Comment:
nit: Conventionally, single-line functions should use the braceless expression form (which needs parens because of the object in this case), e.g.
```suggestion
    _.map(name => ({ value: name, label: name }))
```